spec/set-prototype-is-superset-of.html

```diff
@@ -12,8 +11,7 @@
         1. Let _next_ be IteratorStep(_iter_).
         1. If _next_ is *false*, return *true*.
         1. Let _nextValue_ be ? IteratorValue(_next_).
-        1. Let _has_ be ? Call(_hasCheck_, _set_, « e »).
+        1. Let _has_ be ? Call(_hasCheck_, _set_, « _nextValue_.[[Value]] »).
```

Comment:
Is the .[[Value]] needed here? Doesn’t the ? / ReturnIfAbrupt unwrap the completion record for you?

Reply:
Fixed! I was mostly going off the Set constructor in the spec which was recently fixed too (https://github.com/tc39/ecma262/pull/1310) :smile: 

Updated all the other methods to do the same too.